packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_set_loan_scheme.test.ts

```diff
@@ -1,89 +1,249 @@
-import { GenesisKeys } from '@defichain/testcontainers'
+import { DeFiDRpcError, GenesisKeys, MasterNodeRegTestContainer } from '@defichain/testcontainers'
 import { getProviders, MockProviders } from '../provider.mock'
 import { P2WPKHTransactionBuilder } from '../../src'
 import { fundEllipticPair, sendTransaction } from '../test.utils'
 import { WIF } from '@defichain/jellyfish-crypto'
 import BigNumber from 'bignumber.js'
-import { LoanMasterNodeRegTestContainer } from './loan_container'
 import { Testing } from '@defichain/jellyfish-testing'
 import { RegTest } from '@defichain/jellyfish-network'
 import { LoanTokenResult } from '@defichain/jellyfish-api-core/src/category/loan'
 import { TokenInfo } from '@defichain/jellyfish-api-core/src/category/token'
 
-const container = new LoanMasterNodeRegTestContainer()
+const container = new MasterNodeRegTestContainer()
 const testing = Testing.create(container)
 
 let providers: MockProviders
 let builder: P2WPKHTransactionBuilder
 
-let loanTokenId: string
-
-beforeAll(async () => {
-  await testing.container.start()
-  await testing.container.waitForWalletCoinbaseMaturity()
-
-  providers = await getProviders(testing.container)
-  providers.setEllipticPair(WIF.asEllipticPair(GenesisKeys[GenesisKeys.length - 1].owner.privKey))
-  builder = new P2WPKHTransactionBuilder(providers.fee, providers.prevout, providers.elliptic, RegTest)
-
-  const oracleId1 = await testing.container.call('appointoracle', [await testing.generateAddress(), [{
-    token: 'Token1',
-    currency: 'USD'
-  }], 1])
-  await testing.generate(1)
-
-  const timestamp1 = Math.floor(new Date().getTime() / 1000)
-  await testing.rpc.oracle.setOracleData(oracleId1, timestamp1, { prices: [{ tokenAmount: '0.5@Token1', currency: 'USD' }] })
-  await testing.generate(1)
-
-  const oracleId2 = await testing.container.call('appointoracle', [await testing.generateAddress(), [{
-    token: 'Token2',
-    currency: 'USD'
-  }], 1])
-  await testing.generate(1)
-
-  const timestamp2 = Math.floor(new Date().getTime() / 1000)
-  await testing.rpc.oracle.setOracleData(oracleId2, timestamp2, { prices: [{ tokenAmount: '0.5@Token2', currency: 'USD' }] })
-  await testing.generate(1)
-
-  loanTokenId = await testing.container.call('setloantoken', [{
-    symbol: 'Token1',
-    name: 'Token1',
-    fixedIntervalPriceId: 'Token1/USD',
-    mintable: true,
-    interest: new BigNumber(0.01)
-  }, []])
-  await testing.generate(1)
-})
+describe('loan.createLoanScheme()', () => {
```

Comment:
Edited wrong file? This file is `upload_loan_token` test, how come changed to `set_loan_scheme`?

Reply:
I read correctly right, need more eyes
![image](https://user-images.githubusercontent.com/44192990/141985965-ec8a23ec-d01e-4e37-878f-58b705dbd314.png)

Reply:
Good catch thanks.